composer.json

```diff
@@ -29,35 +29,35 @@
     ],
     "prefer-stable": true,
     "require": {
-        "php": "^8.0",
+        "php": "^8.3",
```

Comment:
What is goal of PHP version bumping to 8.3?

Reply:
Looks like it's similar to https://github.com/yiisoft/yii-gii/pull/140#discussion_r1984783353

Reply:
Seems, `yiisoft/yii-middleware` should good work with both versions of Yii Router. Suggest use constraint `^3.0 || ^4.0`.